KState-HEP-HTT/SMHTT_Analyzers
# Full Refactor

This is a big refactoring. I can now run limits using Higgs Combine without systematics. Yields agree with SVN, but shapes do not. So, this is the last PR before some major debugging. **Wish me luck.**
- Refactor the directory layout
- Add template makers and plotters including the QCD estimation
- Fix the travis script to test any directory
- Add protection for all the headers that should have been there from the start
- Add some safety to the CL parser for parsing multiple arguments to an option
- Fix overlap of tau classes
- Add more MELA variables to the slim tree
- add reweighting for ggH NNLOPS
